### ISSUE TITLE
About normalization

### ISSUE THREAD

**Issue by mathfinder** [sbarratt/inception-score-pytorch#8]
line 88 normalization parameters are (0.5, 0.5, 0.5), (0.5, 0.5, 0.5).
I check https://pytorch.org/docs/stable/torchvision/models.html.
The pretrained model of InceptionV3 are trained with normalization parameters mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225].
![image](https://user-images.githubusercontent.com/23050243/43033353-8d69eb6e-8cfb-11e8-81a4-ffce4b1b3f17.png)
